src/Installer/appcast.xml

```diff
@@ -8,9 +8,9 @@
 		 <item>
 			<title>Version DEV_VERSION_NUMBER</title>
 						<sparkle:releaseNotesLink>
-							http://downloads.palaso.org/FlexBridge/Beta/ReleaseNotes.md
+							http://downloads.palaso.org/FlexBridge/Alpha/ReleaseNotes.md
 						</sparkle:releaseNotesLink>
-			<enclosure url="http://downloads.palaso.org/FlexBridge/Beta/FLExBridgeInstaller.msi" sparkle:version="DEV_VERSION_NUMBER" type="application/octet-stream" />
+			<enclosure url="http://downloads.palaso.org/FlexBridge/Alpha/FLExBridgeInstaller.msi" sparkle:version="DEV_VERSION_NUMBER" type="application/octet-stream" />
```

Comment:
So only one channel per appcast is supported by NetSparkle. I've made a change to copy this into the UploadFolder so that we can have different appcast.xml for the different release types from this point forward.